opentelemetry-stackdriver/Cargo.toml

```diff
@@ -23,7 +23,7 @@ prost = "0.12"
 prost-types = "0.12"
 thiserror = "1.0.30"
 tonic = { version = "0.11", features = ["gzip", "tls", "transport"] }
-yup-oauth2 = { version = "8.3.3", optional = true }
+yup-oauth2 = { version = "9.0.0", optional = true }
```

Comment:
build at root level fails without this upgrade.
```
error: failed to select a version for the requirement `yup-oauth2 = "^8.3.3"`
candidate versions found which didn't match: 9.0.0, 8.3.2, 8.3.1, ...
```

Reply:
lets do it as a separate PR.